scipy-1.0/paper.tex

```diff
@@ -872,8 +872,8 @@ \subsection*{Cython bindings for BLAS, LAPACK, and special}
 LAPACK\cite{LAPACK} routines for many years. SciPy now additionally
 includes Cython\cite{behnel2011cython} wrappers for
 many BLAS and LAPACK routines (added in 2015) and the special functions 
-provided in the \texttt{scipy.{\allowbreak}special} submodule (added in 2016).  
-These Cython wrappers are available in the modules
+provided in the \texttt{scipy.{\allowbreak}special} subpackage (added in 2016).
```

Comment:
That looks ok I think:

```
In [8]: help(special)
Help on package scipy.special in scipy:

NAME
    scipy.special
```

But the official docs so allow use of (sub) prefix for these of course.